projects/kubernetes-sigs/image-builder/patches/0025-Fix-ensure-iptables-present-for-rhel.patch

```diff
@@ -0,0 +1,34 @@
+From 442ef46609a5ae6d518a5bef9d02568993e05994 Mon Sep 17 00:00:00 2001
+From: Ilya Alekseyev <ilya.alekseyev@nutanix.com>
+Date: Fri, 13 Oct 2023 12:20:42 -0400
+Subject: [PATCH 25/25] Fix ensure iptables present for rhel
+
+---
+ images/capi/ansible/roles/setup/tasks/redhat.yml | 14 ++++++++++++++
+ 1 file changed, 14 insertions(+)
+
+diff --git a/images/capi/ansible/roles/setup/tasks/redhat.yml b/images/capi/ansible/roles/setup/tasks/redhat.yml
+index 345aeaa33..a4bc9d03d 100644
+--- a/images/capi/ansible/roles/setup/tasks/redhat.yml
++++ b/images/capi/ansible/roles/setup/tasks/redhat.yml
+@@ -124,3 +124,17 @@
+     name: "{{ extra_rpms.split() }}"
+     state: present
+     lock_timeout: 60
++
++- name: ensure iptables present for redhat 8
++  yum:
++    name: "iptables"
```

Comment:
I feel like rhel 8 uses nft now?

Reply:
> I feel like rhel 8 uses nft now?

I've checked with rhel 8.8 and there is no iptables-nft package

Reply:
thanks, we double checked as well and you are in fact right!